strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/repository/RepositoryFeatures.java

```diff
@@ -11,5 +13,9 @@
     // Indexing implementation
 
     // List of supported checksums
-
+    
+    public void downloadRemoteIndex(String storageId,
```

Comment:
This should not be a method in the interface. In NuGet, this similar to a feed. In NPM and P2 there will be different things specific to those providers.

Reply:
I think that it should. 
> In NuGet, this similar to a feed. In NPM and P2 there will be different things specific to those providers.

all of then is indexes: 
- lucene for Maven
- OrientDB for Nuget

The interface is abstraction which is downloading "indexes" from some kind of "Remote source" and no matter what the indexes are and from what source.

Reply:
Yes, these are indexes in the case of Lucene and OrientDB. However, for NuGet, they're feeds. I'm not sure what they're called in NPM-speak...

This is why I have implemented the concept of repository features.

We should not assume they're interchangeable terms and that each layout provider will have an index. In this particular case, we're not talking about OrientDB, or the Maven Indexer. These are, in effect different concepts.

Reply:
In this case, I'm not sure that I fully understand for now this Repository Features concept, but I will remove method from interface as you wished.

Reply:
Should I leave the method in `NugetRepositoryFeatures` class or move it somewhere else?